vaccine_feed_ingest/utils/match.py

```diff
@@ -67,6 +75,17 @@ def is_concordance_similar(
 
         candidate_concordance[authority].add(value)
 
+    # Handle legacy concordance model by copying over the legacy field
+    # ids as though there was a concordance entry for them.
+    for field, authority in LEGACY_CONCORDANCE_MAP.items():
+        if not candidate_props.get(field):
+            continue
+
+        if authority in candidate_concordance:
```

Comment:
It would be interesting to check if the values matched (we could also do this as a one time query in VIAL)

Reply:
Think this is more important as a query in VIAL instead of buried in the matcher logs